<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditPasswordDialog</class>
 <widget class="QDialog" name="EditPasswordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
	width: 100px;
	height: 30px;
    border-radius: 10px;
	background-color: rgb(0, 0, 172);
	color: rgb(255, 255, 255);
	font: bold 10pt &quot;Google Sans&quot;;
	outline: none;
}

QPushButton:hover:!pressed
{
	width: 100px;
	height: 30px;
   	border-radius: 10px;
	background-color: rgb(223, 223, 223);
	color: rgb(0, 0, 172);
	font: bold 10pt &quot;Google Sans&quot;;
	outline: none;
	border: 2px solid rgb(0, 84, 253)
} 

QLineEdit {
	background-color: rgb(255, 255, 255);
	border-radius: 5px;
	font: bold 10pt &quot;Google Sans&quot;;
	padding-left: 5px;
	height: 30px;
}

QLineEdit:focus {
	background-color: rgb(255, 255, 255);
	border-radius: 5px;
	font: bold 10pt &quot;Google Sans&quot;;	
	border: 2px solid rgb(0, 85, 255);
	padding-left: 5px;
	height: 30px;
}

QLabel {
	color: rgb(70, 49, 190);
	font: bold 10pt &quot;Google Sans&quot;;
	border: none;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Mật khẩu cũ</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="oldPassword">
           <property name="minimumSize">
            <size>
             <width>210</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>210</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Mật khẩu mới</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="newPassword">
           <property name="minimumSize">
            <size>
             <width>210</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>210</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Xác nhận mật khẩu mới</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="checkNewPassword">
           <property name="minimumSize">
            <size>
             <width>210</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>210</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>40</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>40</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="okButton">
        <property name="text">
         <string>Đồng ý</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../img/feather/check-square.svg</normaloff>../img/feather/check-square.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exitButton">
        <property name="text">
         <string>Thoát</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../img/feather/x-square.svg</normaloff>../img/feather/x-square.svg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
